Cross.toml

```diff
@@ -0,0 +1,8 @@
+[target.x86_64-unknown-linux-gnu]
+pre-build = [
+    "apt-get update && apt-get -y install pkg-config libudev1 libudev-dev"
+]
+[target.aarch64-unknown-linux-gnu]
+pre-build = [
+    "dpkg --add-architecture arm64 && apt-get update && apt-get -y install pkg-config libudev1:arm64 libudev-dev:arm64"
+]
```

Comment:
This seems very specific to platforms using apt-get/dpkg. I feel like this should be done on the definition of the GitHub Actions workflow, not here.

Also, because we're not going to ship this in binary releases, you don't need to support `cargo cross` in the first place. Just run `cargo build --release` in the workflow please.

Reply:
ok. I re-used what was done for the other step, so I had to cope with the way ``cross`` was working. I've updated to build again the system without cross. I've also removed the aarch64 build, to keep things simple.